manifest.yml

```diff
@@ -152,6 +154,11 @@ instance_groups:
       secureproxy:
         listen_port: 8081
         proxy_port: 8080
+  - name: bpm
+    release: bpm
+    properties:
+      bpm:
```

Comment:
You probably want to remove these properties and add it to the uaa properties here: https://github.com/18F/cg-deploy-opslogin/blob/master/manifest.yml#L37 .  So you'll just be adding bpm field to the uaa properties.